tests/JalalianTest.php

```diff
@@ -128,6 +128,37 @@ public function testGetWeekOfMonth()
 
         $jDate = new Jalalian(1395, 7, 20);
         $this->assertEquals($jDate->getWeekOfMonth(), 4);
+
+        $jDate = new Jalalian(1401, 1, 5);
+        $this->assertEquals($jDate->getWeekOfMonth(), 1);
+
+        $jDate = new Jalalian(1390, 8, 7);
+        $this->assertEquals($jDate->getWeekOfMonth(), 2);
+
+
+        $jDate = new Jalalian(1390, 8, 27);
+        $this->assertEquals($jDate->getWeekOfMonth(), 4);
+
+        $jDate = new Jalalian(1390, 7, 1);
+        $this->assertEquals($jDate->getWeekOfMonth(), 1);
+
+        $jDate = new Jalalian(1390, 7, 2);
+        $this->assertEquals($jDate->getWeekOfMonth(), 2);
+
+        $jDate = new Jalalian(1390, 7, 30);
+        $this->assertEquals($jDate->getWeekOfMonth(), 6);
```

Comment:
How a month has 6 weeks?

Reply:
Find it in the attached picture.

![image](https://user-images.githubusercontent.com/12831959/165085074-6f513a45-e0ca-4d48-a485-d41a8b4e5bf0.png)